### PR TITLE
docs: update github docs and templates

### LINKED ISSUE #749: Update and add github repo docs
## Expected Behavior

GitHub provides the option for templates to differentiate issues opened that we should look into and create separate templates for Bug, Feature, and Maintenance.

The amount of text in the issue and pull request templates should be updated and cleaned up.

We are missing a copy of `CODE_OF_CONDUCT` in our `.github` docs.

### PR DESCRIPTION
**Pull request by ryanchristo**
##  Changes

- add `CODE_OF_CONDUCT` using recommended GitHub template
- update `ISSUE_TEMPLATE` and `PULL_REQUEST_TEMPLATE`

##  Issues

closes #749

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,26 +1,58 @@
...
+Each empty square brackets below is a checkbox. Replace [ ] with [x] to check
+the box after completing the task.
+
+-->
+
+- [ ] I have read [CONTRIBUTING](CONTRIBUTING.md) and [CODE_OF_CONDUCT](CODE_OF_CONDUCT.md)
+- [ ] I have provided as much information as possible and necessary
+- [ ] I have reviewed my own pull request before requesting a review
+- [ ] I have linked any relevant issues in my pull request comments
+- [ ] All required integration and code climate tests are passing
```
Comment (noot) [inline .github/PULL_REQUEST_TEMPLATE.md:45]: i think just saying integration tests or checks is sufficient, but I'm fine with either

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,26 +1,58 @@
...
+<!--
+
+Each empty square brackets below is a checkbox. Replace [ ] with [x] to check
+the box after completing the task.
+
+-->
+
+- [ ] I have read [CONTRIBUTING](CONTRIBUTING.md) and [CODE_OF_CONDUCT](CODE_OF_CONDUCT.md)
+- [ ] I have provided as much information as possible and necessary
+- [ ] I have reviewed my own pull request before requesting a review
```
Comment (noot) [inline .github/PULL_REQUEST_TEMPLATE.md:43]: maybe mention the diff?

Comment (ryanchristo) [reply .github/PULL_REQUEST_TEMPLATE.md:43]: kk, will add

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -20,36 +14,40 @@ If you're suggesting a change/improvement, tell us how it should work.
...
 -->
 
+-
+
 ## Possible Solution
 
 <!---
 
 Not obligatory, but this is the place to suggest the underlying cause and
 possible fix for the bug, if you have one, or ideas how to implement the
```
Comment (amerameen) [inline .github/ISSUE_TEMPLATE.md:37]: grammer fix: "ideas **on** how to..."

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -20,36 +14,40 @@ If you're suggesting a change/improvement, tell us how it should work.
...
-addition or change. We'll be sure to credit your ideas in the commit log, or
-better yet, submit a PR and you'll get credit for the whole thing.
-
+addition or change. We'll be sure to credit your ideas in the commit log,
```
Comment (amerameen) [inline .github/ISSUE_TEMPLATE.md:38]: suggestion: "addition or change" -> "fix"

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -1,26 +1,58 @@
...
 ## Changes
-- 
-- 
-- 
 
-<!-- Details on how to run only the tests for the changes in the PR -->
+<!--
+
+Please provide a brief but specific list of changes made, describe the change
+in functionality rather than the change in code.
```
Comment (amerameen) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: grammar: "made, describing the changes in functionality rather than the changes in code"

**Pull request merged**